unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BuilderInWorldEntityHandler.cs

```diff
@@ -133,19 +134,31 @@ private void ReportTransform()
         }
     }
 
-    public ParcelScene GetParcelSceneToEdit() { return sceneToEdit; }
+    public ParcelScene GetParcelSceneToEdit()
+    {
+        return sceneToEdit;
+    }
 
-    public List<DCLBuilderInWorldEntity> GetSelectedEntityList() { return selectedEntities; }
+    public List<DCLBuilderInWorldEntity> GetSelectedEntityList()
+    {
+        return selectedEntities;
+    }
 
-    public bool IsAnyEntitySelected() { return selectedEntities.Count > 0; }
+    public bool IsAnyEntitySelected()
+    {
+        return selectedEntities.Count > 0;
+    }
```

Comment:
The new commit hook makes this changes automatically, they will change again after we pass it to the full repository